README.md

```diff
@@ -1 +1,27 @@
-# gapminder_dashboard
\ No newline at end of file
+# Gapminder dashboard
+
+## Decription of the app
+
+Gapminder data set is a great source to know about the world. So The main objective of this app is to investigate what the young population of students know and don’t know about basic global patterns and macro-trends and educate the intended users on global development trends. This dashboard has been developed to communicate insights about historical social/economic trends and create a better understanding of using and teaching statistics.
```

Comment:
The Gapminder dataset is a great source of information to learn about the world. The main objective of this app is to investigate what the young population of students know and don’t know about basic global patterns and macro-trends and educate the intended users on global development trends. This dashboard has been developed to communicate insights about historical social/economic trends and create a better understanding of using and teaching statistics.